js/src/events/Dashboard.js

```diff
@@ -337,6 +337,12 @@ export class EventsDetails extends RenderDetails {
             },
             render: (v, item) => <ImageThumbnail image={v} alt={item.name} image_type="main" width={150}/>,
           },
+          youtube_video_id: {
+            title: 'Youtube video ID',
+            render: (vid) => vid ?
+              <a target="_blank" rel="noopener noreferrer" href={'https://www.youtube.com/watch?v={vid}'}>https://www.youtube.com/watch?v={vid}</a>
```

Comment:
```suggestion
              <a target="_blank" rel="noopener noreferrer" href={`https://www.youtube.com/watch?v=${vid}`'}>https://www.youtube.com/watch?v={vid}</a>
```

Maybe I'm missing something, but doesn't it need to be like this?

Reply:
JSX parses that but I've changed it to stick to convention